modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/resources/com/liferay/dynamic/data/mapping/dependencies/upgrade.v5_3_2/ddm-template-browser-sniffer-content.ftl

```diff
@@ -0,0 +1,25 @@
+<#--
+Web content templates are used to lay out the fields defined in a web
+content structure.
+
+Please use the left panel to quickly add commonly used variables.
+Autocomplete is also available and can be invoked by typing "${".
+-->
+
+<#assign themeId = serviceLocator.findService("com.liferal.kernel.layout.ThemeId") />
+
+<#assign browserSniffer = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil") />
+
+<#assign browserSniffer = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil") >
+
+<#assign
+    browserSniffer = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil")
+    variableA = "valirable A"
+    themeId = serviceLocator.findService("com.liferal.kernel.layout.ThemeId")
+/>
+
+<#assign
+    browserSniffer = serviceLocator.findService("com.liferay.portal.kernel.servlet.BrowserSnifferUtil")
+    variableAB = "valirable B"
+    themeId = serviceLocator.findService("com.liferal.kernel.layout.ThemeId")
+>
```

Comment:
@alberttgomes, why duplicate these lines? Is it really necessary?

Reply:
In this case I'm testing differents types of the syntax, I can create new case test, but I think which not is necessary. In this cenarious, I'm testing this cases: < /> and < >.